code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -439,17 +439,15 @@ DEFINE_BITFIELD(turret_flags, list(
 			if(ispAI(A))
 				continue
 
-			if((turret_flags & TURRET_FLAG_SHOOT_BORGS) && sillycone.stat != DEAD && iscyborg(sillycone))
-				targets += sillycone
-				continue
-
-			if(sillycone.stat || in_faction(sillycone))
-				continue
-
 			if(iscyborg(sillycone))
 				var/mob/living/silicon/robot/sillyconerobot = A
-				if(LAZYLEN(faction) && (ROLE_SYNDICATE in faction) && sillyconerobot.emagged == TRUE)
+				if((turret_flags & TURRET_FLAG_SHOOT_BORGS) && sillyconerobot.stat != DEAD)
+					targets += sillyconerobot
+					continue
+				if(sillyconerobot.stat || in_faction(sillyconerobot))
 					continue
+				if((ROLE_SYNDICATE in faction) && !sillyconerobot.emagged && LAZYLEN(faction))
```

Comment:
```suggestion
				if((ROLE_SYNDICATE in faction) && !sillyconerobot.emagged)
```
`X in null` should still be false, no? No need to check whether the list is populated.

Reply:
That's what I did at first but JohnWillard told to include the LAZYLEN(faction) when he reviewed this. I hate him